src/mlpack/methods/ann/loss_functions/mean_squared_error_impl.hpp

```diff
@@ -31,7 +32,13 @@ MeanSquaredError<InputDataType, OutputDataType>::Forward(
     const PredictionType& prediction,
     const TargetType& target)
 {
-  return arma::accu(arma::square(prediction - target)) / target.n_cols;
+  typename PredictionType::elem_type lossSum =
+    arma::accu(arma::square(prediction - target));
```

Comment:
```suggestion
  typename PredictionType::elem_type lossSum =
      arma::accu(arma::square(prediction - target));
```

Should be tabbed twice - https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping.